packaging.conf

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# ---------------------------------------------------------
+CONFOPT=""
+MAKE_DIST="cd backends/ebpf && ./build_libbpf && cd ../.. && cmake . $CONFOPT && make dist"
+NOT_TARGET_DISTRIBUTIONS="lucid precise trusty xenial bionic"   # <<-- Distrubutions which are *not* supported!
+MAINTAINER="Thomas Dreibholz <dreibh@iem.uni-due.de>"
```

Comment:
This seems like a commitment, who will maintain this over the years?

Reply:
Yes, I can help with this.

The packaging files and scripts are particularly also helpful for users to create PPA packages of (possibly customised) versions. Using Ubuntu Launchpad, it is easy to create own PPAs and upload sources packages, to get binary packages built.